### ISSUE TITLE
change tick holds to not repeatedly miss

### ISSUE THREAD

**Issue by shakesoda** [stepmania/stepmania#1799]
repeatedly missing tick holds is outright unfair on bad pads or with any kind of leg injury.

getting real sick of lifebar failing everything if i don't use no holds, and instantly failing people on missed holds is bad game design anyway

**Comment by coderjo** [comment]
Is this referring to pump-style checkpoint holds? If so, that's how they work in pump.

**Comment by shakesoda** [comment]
@coderjo yes, it's how they work in pump, but it's a bad design and we've got no obligation to respect it.

i haven't committed one way or the other over making the behavior optional, but i'm leaning toward not.
